examples/hello-world/main.c

```diff
@@ -0,0 +1,35 @@
+#include <stdlib.h> // exit
+#include <string.h> // memset
+#include <stdio.h> // fprintf
+#include "../../ui.h"
```

Comment:
```suggestion
#include <ui.h>
```

So this can virtually be dropped into place.
We might need to tell meson to add the top level directory to the include path? Not sure, but this may already be the case.